### ISSUE TITLE
Can't Scaffold a Full Stack App (Cannot find module 'regenerator/runtime')

### ISSUE THREAD

**Issue by javidrashid** [angular-fullstack/generator-angular-fullstack#1518]
Okay so I have the stable node verison v.5.2.0 and I installed the latest yo, grunt-cli as well as bower but once i hit yo angular-fullstack sampleFirstApp, I get the following error, any ideas:

module.js:329
    throw err;
    ^

Error: Cannot find module 'regenerator/runtime'
    at Function.Module._resolveFilename (module.js:327:15)
    at Function.Module._load (module.js:278:25)
    at Module.require (module.js:355:17)
    at require (internal/module.js:13:17)
    at Object.<anonymous> (/usr/local/lib/node_modules/generator-angular-fullstack/node_modules/babel-core/lib/polyfill.js:5:1)
    at Module._compile (module.js:399:26)
    at Object.Module._extensions..js (module.js:406:10)
    at Module.load (module.js:345:32)
    at Function.Module._load (module.js:302:12)
    at Module.require (module.js:355:17)
    at require (internal/module.js:13:17)

**Comment by Awk34** [comment]
Maybe there was a problem installing the generator. Try re-installing it.

**Comment by javidrashid** [comment]
So I re-installed all the components and it looked like it worked but after scaffolding an app and when I run grunt serve for preview the browser would show

Failed to load resource: net::ERR_EMPTY_RESPONSE, which is actually browser has received No Data, I have my mongodb running. Do you know how to get around this?

**Comment by dkmin** [comment]
Same issue here. Please help us.

**Comment by Awk34** [comment]
Do you still have this problem?

**Comment by javidrashid** [comment]
I don't have the same problem as discussed in this thread but a different one maybe manifestation of something related, I got around this issue by reinstalling all the components but now my App never runs, though there are no errors but the localhost:9000 will show a page that tells NO_RESPONSE which means something is wrong with the network settings but actually that is not the case. I tried on different connections, PCs, macbook, using VPS but my app will never load, I think I talked about that issue in a separate thread

https://github.com/angular-fullstack/generator-angular-fullstack/issues/1628

**Comment by Awk34** [comment]
You get this with a freshly-generated app? I don't think I've ever seen that before. Do you have more details?

**Comment by javidrashid** [comment]
Yes, I got away from this problem 

Can't Scaffold a Full Stack App (Cannot find module 'regenerator/runtime')

by reinstalling all the things from scratch, like yo, grunt-cli, bower et-al...

**Comment by Awk34** [comment]
I'm talking about the `ERR_EMPTY_RESPONSE` issue

**Comment by javidrashid** [comment]
I have never been able to get around ERR_EMPTY_RESPONSE. It still exists.

**Comment by Awk34** [comment]
Please share some details

**Comment by javidrashid** [comment]
I have mentioned everything here, Since I don't have any console errors or anything like that, please refer this link, 

https://github.com/angular-fullstack/generator-angular-fullstack/issues/1628

This has all the details. Not sure what else you need.

**Comment by Awk34** [comment]
What happens if you change directories and scaffold out a brand new app, and then run `gulp serve`? Can you see anything with a brand new app?

**Comment by javidrashid** [comment]
I haven't tried it recently since the last issue #1628 was reported but during that time I have tried almost everything, new directories, new app names, using a combinations of selections while generating app, like dropping Socket.io, Mongoose vs Sequel, grunt vs gulp, jade vs html, different authentications and all that and nothing worked for me and I finally gave up on this generator and I found another pretty basic generator bangular. However that said I am going to give it another shot using a latest git pull. 

Rest assured, I did almost everything to get this working but it just didn't; I know it doesn't look like a generator bug most likely, and looks like a network issue

> No data received
> 
> ERR_EMPTY_RESPONSE

but I tried it on 2/3 ISPS, home network, office network, used proxies as well as vpns, mac as well as pc but no luck. That said I am going to give it another shot and will let you know if there is anything specific to report.
